src/components/About.js

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 const About = () => {
 	return (
-  		<div>
-  			<div className="h-header"><h2>Drama Corpora Project</h2></div>
+  		<div className="dracor-page">
+          <h1>Drama Corpora Project</h1>
```

Comment:
Thanks! Would you recommend using ESLint or any other VS Code extension or Prettier configuration to avoid that kind errors in the future.

Reply:
@markschwindt We are using [xo](https://github.com/sindresorhus/xo) for listing in dracor-frontend, so maybe you could use this extension: https://marketplace.visualstudio.com/items?itemName=samverschueren.linter-xo. (Since xo is already part of the project you should not need to `npm install --save-dev xo`)

Reply:
Thanks! Installed, tested --> working great!